pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.usort]
+
+first_party_detection = false
+
+[tool.black]
+
+line-length = 120
+target-version = ["py36"]
+
+[tool.ufmt]
```

Comment:
I think you ended up adding an ignore for the tutorials section on the previous PR - did you remove it on purpose?

Reply:
Yes I did following up a discussion in the internal diff

Reply:
I will provide mode details internally but the idea is to explicitly mute the errors rather than excluding the entire folder